### PR TITLE
Updated Czech translation

### PR DESCRIPTION
**Pull request by Srandista**
Updated Czech translation with story lines (used already existing translations from Adam Rambousek [Diablo 1] and René "Raptor" Spišák [Hellfire])

### CODE REVIEW

File: Translations/cs.po [hunk 1]
```diff
@@ -3647,24 +3633,20 @@ msgstr "Olej Nepropustnosti"
...
-msgstr "{:s} {:s}"
+msgstr ""
 
 #. TRANSLATORS: Constructs item names. Format: {Prefix} {Item} of {Spell}. Example: King's War Staff of Firewall
 #: Source/items.cpp:1193
-#, fuzzy
 msgctxt "spell"
 msgid "{0} {1} of {2}"
-msgstr "{:s} {:s} {:s}"
+msgstr ""
```
Comment (NikoVP) [inline Translations/cs.po:3644]:
```suggestion
msgstr "{0} {1} of {2}"
```

**Comment by AJenbo** [comment]
Thank you :)

**Comment by AJenbo** [comment]
@Psojed @Srandista if either of you have time to update the translation now would be a good time to make it for the next release :)

There are about 250 lines that needs an update.

**Comment by Psojed** [comment]
I'll take a look :)

But on the subject of this commit.. @Srandista do you have permission from the original authors? Or did you just "borrow" these translations from the internet?

**Comment by Srandista** [comment]
@Psojed As stated in OP, I did used already existing traslations. I think it's clearly stated, I take no credit for the transition itself and credited both original authors immediately.

**Comment by Psojed** [comment]
@Srandista I'm not asking where are these translations from. I'm asking, if you had premission from the original authors to use their work.

**Comment by Srandista** [comment]
Yes, I have e-mail confirmation from both authors. Anything else?

**Pull request merged**